py_bind/optv/orientation.pxd

```diff
@@ -31,3 +35,7 @@ cdef extern from "optv/orientation.h":
         int db_length, double db_weight)
 
 cdef calibration** cal_list2arr(list cals)
+
+cdef extern from "optv/epi.h":
```

Comment:
consider putting it in epipolar.pxd and cimporting from it.

Reply:
done